### PR TITLE
Print instead of log what's posted

### PR DESCRIPTION
**Pull request by alistaire47**
Print to stdout so as not to worry about logging across packages. Should solve this better long-term, but for now we need to debug

### CODE REVIEW

**Review by ElenaHenderson — APPROVED** [review]

File: benchadapt/python/benchadapt/client.py [hunk 1]
```diff
@@ -48,8 +48,17 @@ def get(self, path: str) -> dict:
...
+
+        # log.debug(f"POST {url} {dumps(json)}")
+        print(f"POST {url} {dumps(json)}")
+
+        try:
+            res = self.session.post(url=url, json=json, timeout=self.timeout_s)
+        except requests.exceptions.ReadTimeout as e:
+            print(f"POST timed out: {e.response.content.decode()}. Retrying...")
+            print(f"POST {url} {dumps(json)}")
+            res = self.session.post(url=url, json=json, timeout=self.timeout_s)
```
Comment (ElenaHenderson) [inline benchadapt/python/benchadapt/client.py]: Can line 60 please be in another try except block so if it fails, we will continue posting other results in the loop?

We should still raise an error though if the any 2nd try failed but after the loop is done.

Comment (alistaire47) [reply benchadapt/python/benchadapt/client.py]: K, moved all this to the posting loop so we can raise after

**Review by ElenaHenderson — APPROVED** [review]
Thank you!

File: benchadapt/python/benchadapt/adapters/_adapter.py [hunk 2]
```diff
@@ -162,10 +164,23 @@ def post_results(self) -> list:
...
         res_list = []
+        error = None
         for result in self.results:
             result_dict = result.to_publishable_dict()
-            res = client.post(path="/benchmarks/", json=result_dict)
+
+            try:
+                res = client.post(path="/benchmarks/", json=result_dict)
+            except requests.exceptions.ReadTimeout as e:
+                print(f"POST timed out: {e.response.content.decode()}. Retrying...")
```
Comment (ElenaHenderson) [inline benchadapt/python/benchadapt/adapters/_adapter.py]: There will be no response when request times out.

Comment (ElenaHenderson) [reply benchadapt/python/benchadapt/adapters/_adapter.py]: You can print `e` and time it happens so it can be tracked in the conbench logs.

Comment (alistaire47) [reply benchadapt/python/benchadapt/adapters/_adapter.py]: Wait if i use `warnings.warn()` will those end up in the logs? That would be less gross than the `print()`s

**Review by ElenaHenderson — APPROVED** [review]

File: benchadapt/python/benchadapt/adapters/_adapter.py (hunk 2 above)
Comment (ElenaHenderson) [reply benchadapt/python/benchadapt/adapters/_adapter.py]: I really don't know and I never know unless I setup logging for the app myself :)

**Pull request merged**